### DISCUSSION TITLE
Could not find in the doco how to delete a row.

### DISCUSSION THREAD

**Discussion by halloleo** [saulpw/visidata#1323] [Q&A] [answered]
Help! I'm looking to delete/remove the current row from a sheet.

The `DEL` key empties a cell, so I tried `g DEL` and `z DEL`, but that didn't work...

I really couldn't find it, neither in jsvine's tutorial nor in vididata's docs.

**Comment by anjakefala** [comment, marked as answer]
Hi @halloleo

It is `d` for a single row, and `gd` for selected rows! In the menu, it is under `Edit` -> `delete`.

Do you have a page in jsvine's docs where it felt missing? I know VisiData's how to docs need a lot of love from me.

**Reply by halloleo** [reply]
Thanks @anjakefala! 

I now found it in the man page / Quick Ref Guide. Should have looked there myself... (The Quick Ref Guide is actually the best cheatsheet-like page I found for VisiData!)

To your question:

> Do you have a page in jsvine's docs where it felt missing? 

I looked under [Understanding rows](https://jsvine.github.io/intro-to-visidata/basics/understanding-rows/). - It could be mentioned there in a seperate subsection.
